spec/models/measure_condition_permutations/calculator_spec.rb

```diff
@@ -0,0 +1,165 @@
+RSpec.describe MeasureConditionPermutations::Calculator do
+  subject(:calculator) { described_class.new measure_with_conditions }
+
+  let(:measure) do
+    create(:measure)
+      .tap { |m| create :measure_condition, measure_sid: m.measure_sid }
+  end
+
+  describe '#measure_conditions' do
```

Comment:
I think we could move this to be a public scope on another model? It seems to be someone else's api